cwel/style/component/_mini-contact.scss

```diff
@@ -0,0 +1,136 @@
+// Container related markup
+.mini-contact {
+    display: block;
+    padding-top: $s_lg;
+    padding-bottom: $s_lg;
+    text-align: center; // Centering all contents within Mini Contact Component.
+
+    @include breakpoint('md') {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-between;
+    }
+    @include breakpoint('lg') {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-between;
+    }
+
+    &--dual-col {
+        display: block;
+        padding-top: $s_lg;
+        padding-bottom: $s_lg;
+        text-align: center; // Centering all contents within Mini Contact Component.
+
+        @include breakpoint('md') {
+            display: flex;
+            flex-flow: row wrap;
+            justify-content: space-between;
+        }
+    }
+
+    &--single-col {
+        display: block;
+        padding-top: $s_lg;
+        padding-bottom: $s_lg;
+        text-align: center; // Centering all contents within Mini Contact Component.
+
+        @include breakpoint('lg') {
+            display: flex;
+            flex-direction: column;
+            justify-content: space-around;
+        }
+    }
+}
+
+// Grid related markup
+.mini-contact-grid {
+    @include breakpoint('md') {
+        width: grid-cell-size(12);
+    }
+    @include breakpoint('lg') {
+        width: grid-cell-size(4);
+        padding-right: $s_md;
+
+        &:last-child {
+            padding-right: 0;
+        }
+    }
+
+    &--half {
+        @include breakpoint('md') {
+            width: grid-cell-size(12);
+        }
+        @include breakpoint('lg') {
+            width: grid-cell-size(6);
+            padding-right: $s_md;
+
+            &:last-child {
+                padding-right: 0;
+                padding-left: $s_md;
+            }
+        }
+    }
+
+    &--full {
+        @include breakpoint('md') {
+            width: grid-cell-size(12);
+        }
+        @include breakpoint('lg') {
+            width: grid-cell-size(12);
+        }
+    }
+
+    &--alignleft { // Aligning buttons left within in grid element.
+        text-align: left;
+    }
+}
+
+// Styling related markup
+.mini-contact__title {
+    margin-top: 0;
+    margin-bottom: $s_md;
+    color: $white;
+    font-size: 2.8rem;
+
+    @include breakpoint('lg') {
+        text-align: left;
+    }
+
+    &--centered {
+        margin-top: 0;
+        margin-bottom: $s_md;
+        color: $white;
+        font-size: 2.8rem;
+
+        @include breakpoint('lg') {
+            text-align: center;
+        }
+    }
+}
+
+.mini-contact__text {
+    margin-bottom: $s_md;
+    color: $white;
+
+    @include breakpoint('lg') {
+        text-align: left;
+    }
+
+    &--centered {
+        margin-bottom: $s_md;
+        color: $white;
+
+        @include breakpoint('lg') {
+            text-align: center;
+        }
+    }
+}
+
+ul {
```

Comment:
@ambtlv I would get rid of this generic style as these should have been handled by the reset stylesheet. I would just change the lists in mini contact to `p` tags.